AdminUi/src/AdminUi/ClientApp/src/app/utils/odata-response.ts

```diff
@@ -1,3 +1,5 @@
 export interface ODataResponse<Type> {
     value: Type;
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    "@odata.count": number;
```

Comment:
This `@odata.count` is a standard OData response property.
I talked with Daniel A. about this and he said that we already didn't alter the OData response property from `value` to `result` like in the non OData queries, so by altering this `@odata.count` property, are you saying we should alter the entire response?

Reply:
We decided to return `value` from the Backbone, which is okay, because it's an OData standard. 

But that doesn't mean that we should leak that knowledge to the user of the service in the Admin UI. Because imagine we switch to some other protocol in the future. We don't want to update the whole codebase just because of that.

Reply:
I would definitely prefer a unified return type. But that's nothing we should tackle in this PR. So I am fine with how it is right now.